### PR TITLE
Cleanup MI and devops docs

### PR DESCRIPTION
**Pull request by michaellzc**

### CODE REVIEW

**Comment by michaellzc** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3528** <a href="https://app.graphite.dev/github/pr/sourcegraph/handbook/3528" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/sourcegraph/handbook/3528?utm_source=stack-comment).

File: content/departments/product-engineering/engineering/cloud/devops/managed/index.md [hunk 1]
```diff
@@ -95,13 +100,7 @@ The main limitation of this model is that an underlying GCP infrastructure outag
 - **Inbound network access**: The customer may choose between having the deployment be accessible via the public internet and protected by their SSO provider, or for additional security have the deployment restricted to an allowlist of IP addresses only (such as their corporate VPN, etc.)
 - **Outbound network access**: The Sourcegraph deployment will have unfettered egress TCP/ICMP access, and customers will need to allow the
   Sourcegraph deployment to contact their code host. This can be done by having their code-host be publicly accessible, or by allowing the static IP of the Sourcegraph deployment to access their code host.
-- **Cloudflare protections**: The Sourcegraph deployment, if open to the Internet, will be proxied through Cloudflare and leverage security features such as rate limiting and the Cloudflare WAF.
-
-### Access
-
-- To perform the steps outlined in these docs you will need to be a member of:
-  - The google group [gcp-managed](https://groups.google.com/a/sourcegraph.com/g/gcp-managed/members)
-  - 1Password Vaults `Customer managed instances` & `Internal managed instances`
```
Comment (michaellzc) [inline content/departments/product-engineering/engineering/cloud/devops/managed/index.md:104]: this is no longer applicable. being a part of `gcp-devops`  group is sufficient and I remember this group will be provisioned by tech ops by default on their end of onboarding task

access to 1password is moved to onboarding checklist

File: content/departments/product-engineering/engineering/cloud/devops/managed/index.md [hunk 2]
```diff
@@ -118,17 +117,3 @@ All customer credentials, secrets, site configuration, app and user configuratio
...
-
-```
-Error: Error when reading or editing NetworkEndpointGroup: googleapi: Error 400: The network_endpoint_group resource 'projects/sourcegraph-managed-$COMPANY/zones/us-central1-f/networkEndpointGroups/default-neg' is already being used by 'projects/sourcegraph-managed-$COMPANY/global/backendServices/default-backend-service', resourceInUseByAnotherResource
-```
-
-Or similar—this indicates a bug in Terraform where GCP requires an associated resource to be deleted first and Terraform is trying to delete (or create) that resource in the wrong order.
-
-To workaround the issue, locate the resource in GCP yourself and delete it manually and then `terraform apply` again.
-
-[#cloud-devops]: https://sourcegraph.slack.com/archives/C02KX975BDG
```
Comment (michaellzc) [inline content/departments/product-engineering/engineering/cloud/devops/managed/index.md:134]: the homepage for MI should be mostly external facing.

this is operation detail that only we care, so I moved it to the operation guide

File: content/departments/product-engineering/engineering/cloud/devops/managed/index.md [hunk 3]
```diff
@@ -28,7 +28,12 @@ As of 2022-03-10, managed instance is not the recommended deployment method for
 
 See below for the SLAs and Technical implementation details (including Security) related to managed instances.
 
-Please message [#cloud-devops] for any answers or information missing from this page.
+Please message #cloud-devops for any answers or information missing from this page.
+
+When offering customers a Managed Instance, CE and Sales should communicate and gather information for the following topics
+
+- [ ] Customers are comfortable with [security implication](#security) of using a managed instance
+- [ ] Customers's code host should be accessible publically or able to allow incoming traffic from Sourcegraph-owned static IP addresses. (Notes: we do not have proper support for other connectivity methods, e.g. site-to-site VPN)
```
Comment (michaellzc) [inline content/departments/product-engineering/engineering/cloud/devops/managed/index.md:36]: this should address some of the confusion from indeed

**Review by filiphaftek — APPROVED** [review]
👍

**Pull request merged**